docs/user-guide/examples.md

```diff
@@ -371,7 +371,9 @@ deploy:
 
 Something more tricky using `regex`.
 
-In this case a slash is added to `siteexample.io/portainer` and redirect to `siteexample.io/portainer/`. For more details: https://github.com/containous/traefik/issues/563 
+In this case a slash is added to `siteexample.io/portainer` and redirect to `siteexample.io/portainer/`. For more details: https://github.com/containous/traefik/issues/563
```

Comment:
Should this not be `redirected` instead?

Reply:
correct redirected